src/operator/controllers/intents_reconcilers/exp/database.go

```diff
@@ -68,8 +74,16 @@ func (r *DatabaseReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 	}
 
 	if err := r.otterizeClient.ApplyDatabaseIntent(ctx, intentInputList, action); err != nil {
+		errType, errMsg, ok := graphqlclient.GetGraphQLUserError(err)
+		if !ok || errType != graphqlclient.UserErrorTypeAppliedIntentsError {
+			r.RecordWarningEventf(intents, ReasonApplyingDatabaseIntentsFailed, "Failed applying database intents")
```

Comment:
error should be included in the message, like other warnings, e.g. "Failed applying database intents: %s", err.Error()